datastore/schedule-export/README.md

```diff
@@ -0,0 +1,25 @@
+# Scheduling a Cloud Datastore export
+
+This Google App Engine (GAE) app receives export requests at `/cloud-datastore-export` and
+sends an export request to the < href="https://cloud.google.com/datastore/docs/reference/admin/rest/v1/projects/export">Cloud Datastore Admin API</a>. You must give the GAE service account permission to initiate Cloud Datastore exports and
```

Comment:
I think in general this needs a "Before you being" section that explains to the user what is required to go from new project -> deploying this sample.